packages/core/package.json

```diff
@@ -51,6 +51,7 @@
     "prebundle": "1.2.2",
     "rslib": "npm:@rslib/core@0.0.5",
     "rslog": "^1.2.3",
+    "tsconfck": "3.1.3",
```

Comment:
Do we need to precompile it or using the `autoExternal` feature instead? I don't know how to choose between these two now TBH 😂.

Reply:
Got it, tsconfck is an aggressive ESM package without main. I think prebundle should also support `import.meta.resolve()` to resolve ESM entry as well in the future. I also found that rslog is not prebundled, it that on purpose? 🤔

Reply:
Forgot to add rslog before.